models/reports.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+func (rl ReportListingChild) GetKind() string { return rl.Kind }
+func (rl ReportListingChild) GetId() string {
+	if rl.GetKind() == "t1" {
+		return rl.Data.(CommentListingDataChildrenData).Name
+	} else {
```

Comment:
we can get rid of the else and have the second return in the scope of the function